tests/test_c3_to_s3_rolling.py

```diff
@@ -0,0 +1,57 @@
+import io
+from pathlib import Path
+
+import boto3
+import moto
+import pytest
+from scripts.c3_to_s3_rolling import upload_s3_resource
+
+
+@moto.mock_s3
+def test_upload_with_and_without_content_type(local_stac):
+    """
+    Do some simple tests with putting an object on S3 with appropriate content_types.
+    """
+    bucket_name = "fake-bucket"
+    key_json = "path/to/file.json"
+    key_txt = "path/to/file.txt"
+
+    client = boto3.client("s3")
+    client.create_bucket(
+        Bucket=bucket_name,
+        CreateBucketConfiguration={"LocationConstraint": "ap-southeast-2"},
+    )
+
+    content = None
+    with open(local_stac, "r") as f:
+        content = f.read()
```

Comment:
boto3 does still want `bytes` not `str` I think, and this will read a `str`. Try:
```suggestion
    content = local_stac.read_bytes()
```

Reply:
Nah, the test works which means boto works.